BigriverBookstore-apple/Sources/Model/Book.swift

```diff
@@ -8,18 +8,21 @@
 
 import Spine
 
-class Book: Resource {
-    var title: String?
-    var isbn: NSNumber?
+class Book {
```

Comment:
@c-Viorel Same as Author, model classes would be great to be structs. In this way, we block the future temptation to add mutability to models.